modules/ROOT/pages/installing-multiple-instances.adoc

```diff
@@ -0,0 +1,228 @@
+|`proxy.monitoring.proxy` |Monitoring proxy values | socks5://<user>:<pass>@<10.0.0.2>:<8080>
+|`global.containerLogPaths` | Filebeat read path | - /var/lib/docker/ +
+ - /var/log/containers +
+ - /var/log/pods
+|===
+
+[NOTE]
+For the first agent being installed on the cluster, set the value for `crds.install` and `crds.keep` to `true`. +
+Set `crds.install` to `false` for all the subsequent agent installations on the same cluster.
+
+== Values.yml Reference
+The following is an example of the `values.yml` file:
+
+----
+activationData: <activation data>
+  proxy:
+    http_proxy:
+    http_no_proxy:
+    monitoring_proxy:
+custom_log4j_enabled: true
+muleLicense: <mule license key>
+global:
+  crds:
+    install: true
+    keep: true
+  authorizedNamespaces: false
+  image:
+    rtfRegistry: rtf-runtime-registry.kprod.msap.io
+    pullSecretName: rtf-pull-secret
+  containerLogPaths:
+  - /var/lib/docker/containers
+  - /var/log/containers
+  - /var/log/pods
+----
+
+
+== Deploy Mule Applications
+
+. To deploy Mule applications, create an `app-namespace` for each of installed agent: 
++
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: <app-namespace>
+  labels:
+    rtf.mulesoft.com/agentNamespace: <rtf_namespace>
+    rtf.mulesoft.com/envId: <ENVIRONMENT-ID>
+    rtf.mulesoft.com/org: <ORG-ID>
+    rtf.mulesoft.com/role: workers
+----
+[start=2]
+. Create the ingress in the new `rtf_namespace`: +
+----
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: rtf-ingress
+  namespace: <rtf_namespace>
+  annotations:
+    nginx.ingress.kubernetes.io/ssl-redirect: "false"
+    nginx.ingress.kubernetes.io/rewrite-target: /$2
+spec:
+  ingressClassName: rtf-nginx
+  rules:
+    - host: "testrtf.com"
+      http:
+        paths:
+          - pathType: Prefix
+            path: "/app-name(/|$)(.*)"
+            backend:
+              service:    
+                name: service
+                port:
+                  name: service-port
+----
+
+Use a different host name per `rtf_namespace`. If multiple ingresses define different paths for the same host, the ingress controller merges the definitions.  As a result, Mule applications with the same name are not accessible, which causes a k8s issue, not a Runtime Fabric issue.
+
+[[configure-namespaces]]
+== (Optional) Configure Authorized Namespaces
+You can optionally configure authorized namespaces, which enables you to deploy Runtime Fabric alongside other services in a Kubernetes cluster.
+
+You must create the `authorized-namespaces` ConfigMap file before installing Runtime Fabric for the Runtime Fabric namespace. Additionally, you must name the `ConfigMap`, `authorized-namespaces`. The following example shows a `ConfigMap` file:
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: authorized-namespaces
+  namespace: <rtf_namespace>
+data:
+  ADDITIONAL_NAMESPACE_1: "additional-namespace1"
+  ADDITIONAL_NAMESPACE_2: "additional-namespace2"
+----
```

Comment:
@digvijaysk19 can you confirm if this is how support wanted additional namespace key-values to be configured? I believe there was a different suggestion made by them.
Can you please help update it here and also the original doc step 5. at https://docs.mulesoft.com/runtime-fabric/latest/install-helm#before-you-begin-2?